src/parser/ASTBuilder.h

```diff
@@ -442,7 +448,7 @@ class SyntaxNode {
 
 private:
     ASTNodeType m_nodeType : 16;
-    bool m_flag : 1;
+    int m_flag : 8;
```

Comment:
How about using enum value to clearly represent each flag here?

Reply:
That's good idea!

Reply:
I updated my pr